src/model/operations.py

```diff
@@ -18,11 +18,11 @@
 from collections import Counter
 
 import gnomic
-from cameo import models, phenotypic_phase_plane
+from cameo import phenotypic_phase_plane
```

Comment:
You can import production envelope from cobrapy but maybe keep this for now since the plotting is a little nicer.